doc/source/examples/incompressible-flow/2d-mms/2d-mms.rst

```diff
@@ -0,0 +1,251 @@
+- Files to run the different simulations: ``mms_quad/launch_cases.py`` and ``mms_simplex/launch_cases.py``
+- File to reorganize the output of the triangular mesh in a manner that is similar to the quadrilateral case: ``mms_simplex/organize_output.py``. After running the code, a directory ``mms_simplex/initial_output`` is created where the initial simulations output is stored
+- Python script to plot the error: ``plot_error.py``
+
+
+-----------------------
+Description of the Case
+-----------------------
+
+Numerical models requires both verification and validation. While code verification ensures that the mathematical and numerical scheme are consistent and that the code implementation is free from programming errors, code validation is concerned with verifying the agreement of the mathematical model with the underlying physics. The Method of Manufactured Solution  (MMS, see Roache (2002) [#Roache2002]_) is employed for code verification and involves evaluating the error defined as the difference between the manufactured solution  and the computed solution. 
+Obviously, few exact analytical solution of the incompressible Navier-Stokes (NS) equations are known. Therefore, the following procedure is adopted:
+
+- The manufactured solution is chosen such that all terms of the NS equations are non-trivial and non-singular. In other terms, the manufactured solution must be at least twice differentiable, with both its first and second derivatives being non-trivial. The manufactured solution can vary in complexity.
+- Once the manufactured solution is defined, it is substituted into the steady incompressible NS equations, leading to the appearance of source terms. Hence, the manufactuerd solution is the exact solution of the original equations if the source term is added. 
+- The simulations are run at different mesh resolutions and for different degrees of the polynomial approximations of the velocity and pressure.
+- The order of convergence is calculated by measuring the error between the numerical solution and the manufactured solution.
+
+In the current example, we choose the following analytical solution:
+
+.. math::
+  \mathbf{u} &= 2 \begin{bmatrix}
+    (\sin(\pi x))^2 \sin(\pi y) \cos(\pi y)\\
+    \sin(\pi x) \cos(\pi x) (\sin(\pi y))^2 
+   \end{bmatrix}\\
+  p &= \sin(\pi x)\sin(\pi y)
+ 
+The analytical solution is shown in the following figure:
+
+.. image:: Images/analytical_solution.png
+    :alt: The analytical solution: x-component of the velocity (upper-left), y-component of the velocity (upper-right), and pressure (bottom)
+    :align: center
+    :name: analytical_solution
+    :width: 800
+
+Since the velocity function is divergence-free, no source term arises in the continuity equation. Substituting the previous expressions for the velocity and the pressure into the momentum equation leads to the following source term vector:
```

Comment:
```suggestion
Since the velocity function is divergence-free, no source term arises in the continuity equation. Substituting the previous expressions for the velocity and the pressure into the momentum equation leads to the following source term:
```